x/bank/keeper/ERC20Contract.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is the source of this file?
If there is no problem, it would be good to go to types and manage it in the same structure as precomplie's abi file.